src/components/ResourceCard/AddResource/Editors/H5PEditor.js

```diff
@@ -53,12 +53,14 @@ const H5PEditor = (props) => {
         submitAction,
         h5pFile,
       };
+
       handleCreateResourceSubmit(
-        match.params.playlistid,
+        match.params.playlistId,
         resource.newResource.activity.h5pLib,
         resource.newResource.activity.h5pLibType,
         payload,
         resource.newResource.metadata,
+        match.params.projectId,
```

Comment:
Do we need projectId to create activity?

Reply:
this is for redirecting to main page